spark_connect_labextension/app.py

```diff
@@ -0,0 +1,56 @@
+from jupyter_server.extension.application import ExtensionApp
+from spark_connect_labextension.handlers.cluster.start_cluster import StartClusterRouteHandler
+from spark_connect_labextension.handlers.cluster.stop_cluster import StopClusterRouteHandler
+from spark_connect_labextension.handlers.cluster.get_cluster_logs import GetClusterLogRouteHandler
+from spark_connect_labextension.handlers.cluster.get_cluster_error_suggestions import GetClusterErrorSuggestionsRouteHandler
+from spark_connect_labextension.handlers.cluster.get_cluster_status import GetClusterStatusRouteHandler
+from spark_connect_labextension.handlers.cluster.get_clusters import GetClustersRouteHandler
+from spark_connect_labextension.handlers.cluster.get_config_bundles import GetConfigBundlesRouteHandler
+from spark_connect_labextension.handlers.cluster.get_config_options import GetConfigOptionsRouteHandler
+from spark_connect_labextension.handlers.ui_proxy_redirect import SparkUIProxyRedirectHandler
+from spark_connect_labextension.handlers.ui_proxy import SparkUIProxyHandler
+from spark_connect_labextension.config import EXTENSION_ID
+
+
+class SparkConnectExtensionApp(ExtensionApp):
+    """
+    Jupyter Server App for the Spark Connect Labextension
+    """
+    name = "spark_connect_labextension"
+    default_url = f"/{EXTENSION_ID}"
+    # base_url = f"${self.base_url}{EXTENSION_ID}"
+    load_other_extensions = True
+    file_url_prefix = "/render"
+
+    settings = {}
+    handlers = []
+    static_paths = []
+    template_paths = []
+
+    def initialize_settings(self):
```

Comment:
I haven't found a proper docs on how to use Traitlets in this ExtensionApp class. Can you please give me a reference or example on how to do it? I was thinking of using Traitlets but wasn't sure how to do it here.

Reply:
What is special about this class which makes it harder to use traitlets?

Reply:
Nothing, but I don't know how to use Traitlets here yet 😅

Reply:
I believe that since you inherit from `ExtensionApp`, you are already configurable here. So you just need to define the configurable options ([example](https://github.com/jupyter-server/jupyter_server/blob/main/jupyter_server/extension/application.py#L150-L157)) and configure them via jupyter config python files.